opcat-docs/dapp-development/cat-token-integration/guard-variants.md

```diff
@@ -0,0 +1,130 @@
+---
+id: guard-variants
+title: Guard Variants
+sidebar_position: 2
+---
+
+# Guard Variants
+
+## Overview
+
+CAT Protocol provides multiple guard contract variants to balance transaction size and functionality. Each variant is optimized for different use cases, allowing you to choose the right configuration for your application.
+
```

Comment:
Add a section that informs developers that they should know this issue if they intend to develop their application contracts. If they simply utilize the SDK’s feature functions, such as `send`, the guard variants will be automatically handled.

Reply:
Fixed in commit 2c22c39. Added an info box in the Overview section explaining that application contract developers need to understand guard variants, while SDK users can rely on automatic handling via functions like `send`.